pixi.toml

```diff
@@ -70,13 +66,19 @@ sphinx-autodoc-typehints = "*"
 myst-nb = "*"
 econml = ">=0.15"
 doubleml = "*"
+lime = "*"
 statsmodels = "*"
-
+nbconvert = "*"
 [feature.docs.tasks]
+# Note that this task will fail on systems without bash.
+nb-convert = "nb-convert.sh"
+nb-mypy = "mypy docs/examples/*.py"
 docs = "cd docs && make html "
 readthedocs = "rm -rf $READTHEDOCS_OUTPUT/html && cp -r docs/_build/html $READTHEDOCS_OUTPUT/html"
 
 [feature.build.dependencies]
+setuptools-scm = "*"
+setuptools = ">=61"
```

Comment:
these two (and also `pip`) should go into `[host-dependencies]` as they are also required for the regular `postinstall` task and thus need to be in the default envs as well.

Reply:
Ah, I see!

https://github.com/Quantco/metalearners/pull/102/commits/855d91ba558834e84c82ed5521bc3cbd1622e748